src/script/auth/component/EntropyCanvas.tsx

```diff
@@ -98,6 +98,7 @@ const EntropyCanvas = (props: CanvasProps) => {
     const drawPoint: EntropyFrame = {
       x: event.clientX - boundingRect.x,
       y: event.clientY - boundingRect.y,
+      t: 0,
```

Comment:
this seems like an unnecessary addition to satisfy the type. i think maybe we should use a partial instead.

Reply:
or set Date.now() here.

Reply:
Changed it to a separate type, because `EntropyFrame` is actually not what we have here. What do you think?

Reply:
I personally prefer a partial because it wouldn't add an extra type and it would should a relation between the two types. Either way is fine though